tests/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+USER_ID=${LOCAL_USER_ID:-1001}
+
+echo "Starting with UID : $USER_ID"
+useradd --shell /bin/bash -u $USER_ID -o -c "" -m user
```

Comment:
It's always a good idea to make the user part of gid 0 inside Docker.
This is anyway what all users have (even when you run a container with `--user <random_id>`) due to container user being a member of the root group.

Details [here](https://docs.openshift.org/latest/creating_images/guidelines.html#openshift-specific-guidelines)